local-cli/generator-windows/index.js

```diff
@@ -42,7 +43,30 @@ module.exports = yeoman.Base.extend({
   writing: function () {
     const projectGuid = uuid.v4();
     const packageGuid = uuid.v4();
-    const templateVars = { name: this.name, ns: this.options.ns, projectGuid, packageGuid };
+    const currentUser = childProcess.execSync('powershell $env:username')
+    const templateVars = { name: this.name, ns: this.options.ns, certificateThumbprint : null, projectGuid, packageGuid, currentUser };
+
+    const certGenCommand = [
+      `$cert = New-SelfSignedCertificate -KeyUsage DigitalSignature -KeyExportPolicy Exportable -Subject "CN=${currentUser}" -TextExtension @("2.5.29.37={text}1.3.6.1.5.5.7.3.3", "2.5.29.19={text}Subject Type:End Entity") -CertStoreLocation "Cert:\\CurrentUser\\My"`,
+      `$pwd = ConvertTo-SecureString -String password -Force -AsPlainText`,
```

Comment:
Yes. Sadly `Export-PfxCertificate` [cmdlet](https://technet.microsoft.com/en-us/itpro/powershell/windows/pkiclient/export-pfxcertificate) allows exporting to PFX either by providing password or protecting it to certain users under a domain. 
Hence, I choose to use a password.